docs/docs/modeling.md

```diff
@@ -108,6 +107,25 @@ classDiagram
 
 ## Category - カテゴリー
 
+### Category ID
+
+Category ID is an unique ID.
+
+### Name
+
+The name of group to which the task belongs.  
+Kitchen, Bathroom, Entrance and Living room are set by default. The landlord can create additional categories as they like apart from those listed.
+
+#### Constraints
+
+- the name must be unique.
+- It must be less than equal to 15 letters and greater than equal to 1.
+- The maximum number of categories per sharehouse is 20.
```

Comment:
I think the following constraint belongs to the `Category` itself, not just to the `Name`.
- The maximum number of categories per sharehouse is 20.